packages/sanity-suite/public/v3/dataResidency/index-cdn.html

```diff
@@ -0,0 +1,156 @@
+            ]
+          };
+        }
+      };
+      window.OnetrustActiveGroups = ',C0001,C0003,';
+
+      var sdkBaseUrl = "https://cdn.rudderlabs.com/beta/3.0.0-beta";
+      var sdkName = "rsa.min.js";
+      var asyncScript = true;
+      window.rudderAnalyticsBuildType = "legacy";
+      window.rudderanalytics = [];
+      var methods = [ "setDefaultInstanceKey", "load", "ready", "page", "track", "identify", "alias", "group", "reset", "setAnonymousId", "startSession", "endSession" ];
+      for (var i = 0; i < methods.length; i++) {
+        var method = methods[i];
+        window.rudderanalytics[method] = function(methodName) {
+          return function() {
+            window.rudderanalytics.push([ methodName ].concat(Array.prototype.slice.call(arguments)));
+          };
+        }(method);
+      }
+      try {
+        new Function('return import("")');
+        window.rudderAnalyticsBuildType = "modern";
+      } catch (e) {}
+      window.rudderAnalyticsMount = function() {
+        var rudderAnalyticsScript = document.createElement("script");
+        rudderAnalyticsScript.src = "".concat(sdkBaseUrl, "/").concat(window.rudderAnalyticsBuildType, "/").concat(sdkName);
+        rudderAnalyticsScript.async = asyncScript;
+        if (document.head) {
+          document.head.appendChild(rudderAnalyticsScript);
+        } else {
+          document.body.appendChild(rudderAnalyticsScript);
+        }
+      };
+      if (typeof Promise === "undefined") {
+        var rudderAnalyticsPromisesScript = document.createElement("script");
+        rudderAnalyticsPromisesScript.src = "https://polyfill.io/v3/polyfill.min.js?features=globalThis%2CPromise&callback=rudderAnalyticsMount";
+        rudderAnalyticsPromisesScript.async = asyncScript;
+        if (document.head) {
+          document.head.appendChild(rudderAnalyticsPromisesScript);
+        } else {
+          document.body.appendChild(rudderAnalyticsPromisesScript);
+        }
+      } else {
+        window.rudderAnalyticsMount();
+      }
+      // New addition to load script ends
+
+      // Sets the user ID/anonymous ID
+      // Sending these Identify and Track event to test the preload buffer
+      rudderanalytics.setAnonymousId("automationqa_123");
+
+      // Identify call
+      rudderanalytics.identify("12345", {
+        name: "Automation_QA"
+      });
+
+      // Track call
+      rudderanalytics.track("test_event", {
+        testProp: "automationQA_prop1"
+      });
+
+      var loadOptions={
+        logLevel: 'DEBUG',
+        configUrl: '__CONFIG_SERVER_HOST__',
+        destSDKBaseURL: sdkBaseUrl + '/' + window.rudderAnalyticsBuildType + '/js-integrations',
+        pluginsSDKBaseURL: sdkBaseUrl + '/' + window.rudderAnalyticsBuildType + '/plugins',
+        cookieConsentManager: {
+          oneTrust: {
+            enabled: true,
+          },
+        },
+        storage: {
+          migrate: true
+        }
+      };
+
+      // rudderanalytics.load(
```

Comment:
why is this commented out?

Reply:
We call ` load api` at runtime with different `loadoptions` to test all Data Residency test cases.